### PR TITLE
Add base safes

### PR DESCRIPTION
**Pull request by danielpartida**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: models/safe/base/safe_base_singletons.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+                                    "project",
+                                    "safe",
+                                    \'["danielpartida"]\') }}'
+    )
+}}
+
+
+-- Fetch all known singleton/mastercopy addresses used via factories.
+select distinct singleton as address
+from {{ source('gnosis_safe_base', 'GnosisSafeProxyFactoryv_1_3_0_evt_ProxyCreation') }}
```
Comment (Hosuke) [inline models/safe/base/safe_base_singletons.sql:16]: The source table name is tricky here, there is a letter `v` after `Factory`:
https://dune.com/queries/2965816?category=decoded_project&blockchain=base&namespace=gnosis_safe&contract=GnosisSafeProxyFactoryv_1_3_0&blockchains=base

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @danielpartida 

- Partition by block_month ✅
- Dummy legacy models added for new spells ✅

[Sample safe_base_safes](https://dune.com/queries/2965954)
[Sample safe_base_transactions](https://dune.com/queries/2965957)
[Sample safe_base_eth_transfers](https://dune.com/queries/2965962)

**Comment by danielpartida** [comment]
You are a rockstar @Hosuke ! LFG!

**Comment by danielpartida** [comment]
Do you see any additional blockers @Hosuke 
wen merge?

**Comment by Hosuke** [comment]
> Do you see any additional blockers @Hosuke wen merge?

I will pass to Jeff for the final review.

**Comment by danielpartida** [comment]
@jeff-dude let me know if further input is necessary from my end

File: models/safe/base/safe_base_singletons.sql (hunk 1 above)
Comment (jeff-dude) [reply models/safe/base/safe_base_singletons.sql:16]: likely missed `_` between the `Factory` and `_1_3_0` in the decoding process

if ever fixed there, we'll fix here, but this works for now

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
> @jeff-dude let me know if further input is necessary from my end

all is good, getting a batch together to merge now

**Pull request merged**